openml-lightgbm-provider/pom.xml

```diff
@@ -0,0 +1,195 @@
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
+                        <LC_ALL>C</LC_ALL>
+                    </environmentVariables>
+                </configuration>
+            </plugin>
+
+            <!-- Install the local copy of the custom build of LightGBM to Maven -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-install-plugin</artifactId>
+                <version>3.0.0-M1</version>
+                <executions>
+                    <execution>
+                        <id>install-custom-lightgbmlib_build</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>install-file</goal>
+                        </goals>
+                        <configuration>
+                            <file>${project.basedir}/lightgbmlib_build/lightgbmlib.jar</file>
+                            <pomFile>${project.basedir}/lightgbmlib_build/pom.xml</pomFile>
+                            <generatePom>false</generatePom>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- Copy the shared libraries to the resources folder -->
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.1.0</version>
+                <executions>
+                    <execution>
+                        <id>copy-shared-libs-resources</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.basedir}/target/resources</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>lightgbmlib_build</directory>
+                                    <include>*.so</include>
+                                    <include>*.so.1.0.0</include>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>copy-licences</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                          <goal>copy-resources</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.basedir}/target/resources</outputDirectory>
+                            <resources>
+                                <resource>
+                                    <directory>${maven.multiModuleProjectDirectory}/LICENCES</directory>
```

Comment:
You're missing the licence files. Check the ones which make sense to be copied here.

Reply:
I will change to `<directory>${project.parent.basedir}/LICENSE</directory>`
which is the file that I have in parent.